apps/heureka/src/components/common/IssuesCountPerSeverityLevel.tsx

```diff
@@ -20,22 +20,30 @@ type IssuesCountPerSeverityLevelProps = {
   displayMode?: "all" | "criticalHigh"
 }
 
-type ToolTipBadgeType = {
+type IssuesCountBadgeType = {
   icon?: KnownIcons
-  text: string
+  count: number
   variant: BadgeVariantType
   tooltipContent: string
   className?: string
 }
 
-const ToolTipBadge = ({ icon, text, variant, tooltipContent, className }: ToolTipBadgeType) => (
-  <Tooltip triggerEvent="hover">
-    <TooltipTrigger>
-      <Badge icon={icon} text={text} variant={variant} className={className} />
-    </TooltipTrigger>
-    <TooltipContent>{tooltipContent}</TooltipContent>
-  </Tooltip>
-)
+export const IssueCountBadge = ({ icon, count, variant, tooltipContent, className }: IssuesCountBadgeType) =>
```

Comment:
I'd still call it `SeverityCount` instead of `IssueCountBadge` because it shows count for a given severity level and we don't need to put `Badge` in name because that is just a representation which does not need to be part of the component name. 
Moreover I'll move this component to a separate file under `common` because this is being used across multiple components.